package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodebot-workshop",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
Please remove this, the version will be bumped during the next `npm version && npm publish`

Reply:
Reverted this part.